lib/interactive.dart

```diff
@@ -194,9 +198,53 @@ class Interactive {
       peopleByLabel[key].add(people);
     });
 
+    for (var i = 0; i < xAxisCategories.length; ++i) {
+      var colSVG = svg.SvgElement.tag("g");
+
+      var colData = peopleByLabel[xAxisCategories[i].value];
+      num colOffset = (i + 0.5) * (chartWidth / xAxisCategories.length);
+
+      for (var j = 0; j < colData.length; ++j) {
+        num x = (j % SQ_IN_ROW - SQ_IN_ROW / 2) * SQ_SIZE + colOffset;
+        num y = (CHART_HEIGHT - CHART_XAXIS_HEIGHT - 1.5 * SQ_SIZE) -
+            (j / SQ_IN_ROW).truncate() * SQ_SIZE;
+        num xOrigin = x - 1.5 * SQ_SIZE;
+        num yOrigin = y - 1.5 * SQ_SIZE;
+
+        var sqGroup = svg.SvgElement.tag("g")
+          ..setAttribute("transform-origin", "$xOrigin $yOrigin")
+          ..onMouseEnter
+              .listen((e) => this.handleMouseEnter(e.currentTarget, SQ_SIZE))
+          ..onMouseOut.listen((e) => this.handleMouseOut(e.currentTarget));
+
+        var square = svg.RectElement()
+          ..setAttribute("x", x.toString())
+          ..setAttribute("y", y.toString())
+          ..setAttribute("width", SQ_SIZE.toString())
+          ..setAttribute("height", SQ_SIZE.toString())
+          ..setAttribute("fill", "black")
+          ..setAttribute("stroke", "white")
+          ..setAttribute("stroke-width", "2");
+        sqGroup.append(square);
+
+        colSVG.append(sqGroup);
+      }
+
+      svgContainer.append(colSVG);
+    }
+
     return chartWrapper..append(svgContainer);
   }
 
+  void handleMouseEnter(svg.SvgElement rect, int w) {
+    rect.parent.append(rect);
+    rect..setAttribute("transform", "translate(${-2 * w}, ${-2 * w}) scale(2)");
+  }
+
+  void handleMouseOut(svg.SvgElement rect) {
+    rect..setAttribute("transform", "translate(0, 0) scale(1)");
+  }
+
   html.DivElement _renderLegend() {
     var legendWrapper = html.DivElement()..classes = [BS_ROW_CSS];
```

Comment:
What does `BS_ROW_CSS` mean?

Reply:
Bootstrap row css - removed the prefix, fixed